src/state/store.ts

```diff
@@ -1,5 +1,7 @@
 import { configureStore } from '@reduxjs/toolkit'
+import localforage from 'localforage'
 import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux'
+import { persistReducer, persistStore } from 'redux-persist'
```

Comment:
```suggestion
import { PERSIST, persistReducer, persistStore } from 'redux-persist'
```

https://github.com/rt2zz/redux-persist/blob/57d1c6ccd37dd8dcfd30f6485e6b35baf5a8f51f/src/constants.ts#L5